Src/AutoFixture/Kernel/TypeExtensions.cs

```diff
@@ -8,6 +8,21 @@
 
     internal static class TypeExtensions
     {
+        public static TAttribute GetRequestAttribute<TAttribute>(object request) where TAttribute : Attribute
+        {
+            var member = request as MemberInfo;
+            if(member != null)
+            {
+                return member.GetCustomAttribute<TAttribute>();
+            }
+            var parameter = request as ParameterInfo;
+            if(parameter != null)
+            {
+                return parameter.GetCustomAttribute<TAttribute>();
+            }

```

Comment:
> Type is MemberInfo

Thank you for reminding me; I'd completely forgotten that. Not particularly intuitive, that inheritance hierarchy...